app/javascript/src/components/Dashboard/Notes/List/index.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState, useEffect } from "react";
+
+import EmptyNotesListImage from "images/EmptyNotesList";
+import { Button, PageLoader, Typography } from "neetoui/v2";
+import { Container, Header, SubHeader, MenuBar, Scrollable } from "@bigbinary/neetoui/v2/layouts";
+
+
+import notesApi from "apis/notes";
+import EmptyState from "components/Common/EmptyState";
+
+import DeleteAlert from "./DeleteAlert";
+import Note from "./Note";
+import FilterBar from "./FilterBar";
+
+import NewNotePane from "../Pane/CreateNote";
```

Comment:
We can name both the component and file `NewNote` and `NewNote.jsx` or `Create.jsx`. Let's refrain from prefixing or suffixing file, folder, component names with `Modal, Pane, Form, Page` etc.